build_push_docker.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# This script is used to build and deploy docker images for Picard
+
+if [[ "$1" == "" ]]
+then
+    echo "Usage: build_push_docker.sh <git-tag>"
+    exit 1
+fi
+
+declare -r TAG=${1}
+
+echo "Will build and push the following docker images:"
+echo "broadinstitute/picard:${TAG}"
+echo "broadinstitute/picard-cloud:${TAG}"
+
+read -p "Is this really what you want to do? " -n 1 -r
+echo    # (optional) move to a new line
+if [[ $REPLY =~ ^[Yy]$ ]]
+then
+    docker build -t broadinstitute/picard:${TAG} --build-arg build_command=shadowJar --build-arg jar_name=picard.jar .
+    docker build -t broadinstitute/picard-cloud:${TAG} --build-arg build_command=cloudJar --build-arg jar_name=picardcloud.jar .
```

Comment:
Bike-shedding: I wonder if it should be `/picard:${TAG}-cloud` instead of `/picard-cloud:${TAG}`

Reply:
I think we should be keeping the two versions in different repos since they do different things. The automated `latest` build would be ambiguous as to which tool it refers to.